diffkemp/llvm_ir/compiler.py

```diff
@@ -0,0 +1,46 @@
+"""
+Functions for compilation of c files to LLVM IR, optimizations of LLVM IR.
+
+Syntax of this module must be python2 compatible because it is checked
+when building diffkemp.bulding.cc_wrapper with RPython.
+"""
+from diffkemp.utils import get_opt_command
```

Comment:
Could we move `get_opt_command` to `compiler.py`? I'd like to minimize the amount of modules which must be Python2-compatible.

Reply:
- I could move `get_opt_command` to `compiler.py`, but it would be necessary to also move `get_llvm_version` to `compiler.py` because it is called from `get_opt_command`, then the `utils.py` syntax would not have to be python2 compatible.
- The second option which comes to my mind, is to move the `opt_llvm` function from `compiler.py` to a different file (eg. `diffkemp/llvm_ir/optimiser.py`), because it is not used from `cc_wrapper` (by `rpython`), only the `get_clang_default_options` is used from `cc_wrapper`, but rpython requires all functions in the file and called from them to have python2 syntax. This would minimise the functions which would have to have python2 syntax, but there would be two new modules/files (`compiler.py` and eg. `optimiser.py`) which would both contain only one function for now.

Let me know, which option would you prefer.

Reply:
Option 1 doesn't make much sense to me as `get_llvm_version` is used from other modules, too.

So, I'd either go with option 2 or with moving `get_opt_command` to `compiler.py` and moving `get_llvm_version` to a separate file (`llvm-version.py`?). I don't really have a strong preference here.

Reply:
I used option 2 (moved the `opt_llvm` to `optimiser.py` ), problem with the third option was that the `get_opt_command` is also used from multiple modules.